packages/help-center/src/components/help-center-content.tsx

```diff
@@ -35,12 +37,13 @@ const HelpCenterContent: React.FC< Content > = ( { isMinimized } ) => {
 	}, [ location, section ] );
 
 	return (
-		<CardBody hidden={ isMinimized } className={ className }>
+		<CardBody hidden={ isMinimized } className={ className } ref={ containerRef }>
 			<Route exact path="/">
 				<HelpCenterSearch />
 			</Route>
 			<Route path="/post">
 				<HelpCenterEmbedResult />
+				<BackToTopButton container={ containerRef } />
```

Comment:
I feel like `help-center-content` should remain more routing focused and not on structure. Would it be possible to move the `BackToTopButton` inside `HelpCenterEmbedResult` ?

Reply:
Makes sense to me.
Moved it to `HelpCenterEmbedResult` now.